packages/api/endpoints/granule-csv.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const { Parser } = require('json2csv');
+const router = require('express-promise-router')();
+
+const { Granule } = require('../models');
+
+/**
+ * Builds a CSV file of all granules in the Cumulus DB
+ *
+ * @param {Object} req - express request object
+ * @param {Object} res - express response object
+ * @returns {Object} the csv file of granules
+ */
+async function list(req, res) {
+  const granuleModel = new Granule();
+  const allDbGranules = await granuleModel.scan();
```

Comment:
This could get expensive if a user has millions of granules because DyanmoDB scan. I don't think that's bad, but would it make sense to have some kind of warning/info to notify a user? That could easily be out of scope and another ticket.